lms/envs/common.py

```diff
@@ -2803,7 +2803,7 @@
 ANALYTICS_DATA_URL = ""
 ANALYTICS_DATA_TOKEN = ""
 ANALYTICS_DASHBOARD_URL = ""
-ANALYTICS_DASHBOARD_NAME = PLATFORM_NAME + " Insights"
+ANALYTICS_DASHBOARD_NAME = _('edX Insights')
```

Comment:
Hey @UmanShahzad, this won't work because of the following:

 - `_(PLATFORM_NAME + ' Insights')` won't make it right into `gettext` catalogue ([ref](https://inventwithpython.com/blog/2014/12/20/translate-your-python-3-program-with-the-gettext-module/)). 
 - The `ANALYTICS_DASHBOARD_NAME` is still backward compatible thus your use case is still perfectly supported. If you pass it using the TOKEN_ENVS (which is the default behaviour in `configuration`) you'll get your value. I'll tag you in the `lms/envs/aws.py`.